selfdrive/car/toyota/carstate.py

```diff
@@ -94,6 +97,8 @@ def __init__(self, CP):
     self.shifter_values = self.can_define.dv["GEAR_PACKET"]['GEAR']
     self.left_blinker_on = 0
     self.right_blinker_on = 0
+    self.offset = 0
```

Comment:
Ideally I'd use a bool, but I'm lazy.

Reply:
That's the actual numerical value, which should have a resolution of at least 0.1 degrees

Reply:
Ah, I have misread this. I was thinking of my isoffset=0, which would be a bool instead of an int. 

Thanks for that feedback! It's likely why I had a 1 degree+/- offset since building my steering angle sensor (which didn't impact performance nearly as much as I thought it would, now that I know)